From 822fdc3556b688103cdaf7b4b34e98fbe1676425 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sun, 7 Aug 2022 10:58:02 -0500
Subject: [PATCH 092/117] dt-bindings: display: sun4i-tcon: Add external LVDS
 PHY

A100 and D1 use the same "combo" PHY for LVDS0 and DSI.

Signed-off-by: Samuel Holland <samuel@sholland.org>
---
 .../bindings/display/allwinner,sun4i-a10-tcon.yaml         | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tcon.yaml
+++ b/Documentation/devicetree/bindings/display/allwinner,sun4i-a10-tcon.yaml
@@ -80,6 +80,13 @@ properties:
   dmas:
     maxItems: 1
 
+  phys:
+    maxItems: 1
+
+  phy-names:
+    items:
+      - const: "lvds0"
+
   resets:
     anyOf:
       - items:
